src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1555,6 +1559,29 @@ impl Coordinator {
             }
         }
 
+        for id in &ids {
+            match id {
+                // Validate that `ClusterReplica` drops do not drop replicas of managed clusters,
+                // unless they are with specific billing information, which exist outside the scope
+                // of managed clusters.
```

Comment:
How did this work before? Where were we validating that managed replicas were not being dropped?

Reply:
Yes, during planning we prevented access to the `DROP CLUSTER REPLICA` syntax for managed clusters.